apps/browser-extension-wallet/src/views/bitcoin-mode/features/send/components/SendStepOne.tsx

```diff
@@ -230,10 +241,19 @@ export const SendStepOne: React.FC<SendStepOneProps> = ({
                 onChange={(e) => {
                   setCustomFee(e.target.value);
                 }}
-                onKeyDown={(e) => {
-                  const target = e.target as HTMLInputElement;
-                  if (e.key === '-' || e.key === 'e' || (e.key === 'ArrowDown' && Number(target.value) <= 0)) {
-                    e.preventDefault();
+                onKeyDown={handleCustomFeeKeyDown}
+                onWheel={(e) => {
+                  e.preventDefault();
+
+                  const step = CUSTOM_FEE_STEP;
+                  const currentValue = Number(customFee);
+
+                  // Handle scrolling up (increment) or down (decrement but not below 0)
+                  const newValue = e.deltaY < 0 ? currentValue + step : Math.max(0, currentValue - step);
```

Comment:
The check for positive value happens bellow, so we could skip this one.
```suggestion
                  const newValue = e.deltaY < 0 ? currentValue + step : currentValue - step;
```

Alternatively (more concise, not sure if readable enough):
```suggestion
                  const newValue = currentValue + (step * e.deltaY < 0 -1 : 1);
```